StringPermutation.java

```diff
@@ -0,0 +1,69 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by ioana-chirca on 16-May-17.
+ */
+public class StringPermutation {
```

Comment:
Have you tested it for the example input from the assignment text?
Listen, Silent ⇒ True
Triangle, Integral ⇒ True
Apple, Pabble ⇒ False
Does it return correct values?